src/components/apps/launch/AppLaunchWizard.js

```diff
@@ -25,25 +28,33 @@ function AppLaunchWizard(props) {
             deprecatedParamTypes.includes(param.type)
         )
     );
+    const appInfoRef = React.useRef(null);
+    const [appInfoHeight, setAppInfoRef] = useComponentHeight();
+
+    React.useEffect(() => {
+        setAppInfoRef(appInfoRef);
+    }, [appInfoRef, setAppInfoRef]);
```

Comment:
This useEffect should also not be needed

Reply:
Sorry, these comments weren't super clear - you still need to do `setAppInfoRef(appInfoRef)`, but that line doesn't need to be within a useEffect, is what I meant.